portfolio/src/test/java/com/google/sps/ArrayListTimeRangeGroupTest.java

```diff
@@ -169,6 +169,34 @@ public void testAddOneOverlappingRange() {
     Assert.assertEquals(expected, actual);
   }
 
+  /** Testing for adding the exact same time range. */
+  @Test
+  public void testAddExactSameRange() {
```

Comment:
I think we now have TimeRangeGroupTest [1] that tests both ArrayListTimeRangeGroup and LinkedListTimeRangeGroup. Can we add this test there, and maybe just remove ArrayListTimeRangeGroupTest.java entirely? But if you do so please make sure we port over any test that is in ArrayListTimeRangeGroupTest but not in TimeRangeGroupTest.

[1] https://github.com/googleinterns/step61-2020/blob/master/portfolio/src/test/java/com/google/sps/TimeRangeGroupTest.java

Reply:
Updated.